tests/manager-regression-multiDC.yaml

```diff
@@ -1,47 +1,41 @@
-test_duration: 10080
-stress_cmd: cassandra-stress write cl=QUORUM duration=10080m -schema 'replication(strategy=NetworkTopologyStrategy,us-east1-b=3,us-west1-b=3,us-east4-b=2)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..100000000000
-cassandra_stress_duration: 10080
-cassandra_stress_threads: 1000
-cassandra_stress_population_size: 100000000000
-#n_db_nodes: 3
-n_db_nodes: '5 5 3'
-n_loaders: 3
+test_duration: 2860
+stress_cmd: "cassandra-stress write cl=QUORUM duration=2860m -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=2,us-west-2scylla_node_west=1)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=400200300..600200300"
```

Comment:
replace duration=2860m with n=1200300.
You just want to create some data, and you don't need it to last too long.
This will create 1M keys and should take much less time.